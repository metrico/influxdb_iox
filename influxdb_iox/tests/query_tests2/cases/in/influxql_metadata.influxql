-- IOX_SETUP: InfluxQLSelectSupport

-- SHOW TAG VALUES
SHOW TAG VALUES WITH KEY = "tag0";
