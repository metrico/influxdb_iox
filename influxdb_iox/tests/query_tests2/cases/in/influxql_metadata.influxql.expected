-- Test Setup: InfluxQLSelectSupport
-- InfluxQL: SHOW TAG VALUES WITH KEY = "tag0";
name: select_test
+------+-------+
| key  | value |
+------+-------+
| tag0 | a     |
| tag0 | b     |
+------+-------+
